### DISCUSSION TITLE
Cross-Platform Control and Syncing.

### DISCUSSION THREAD

**Discussion by FerasK0** [pear-devs/pear-desktop#780] [Ideas]
As The title says, I don't know about Google's API. 
But if it's possible it'd be very cool to have a feature where you can control the music on your phone through your PC using YTM API.
there's an independent app (much like Spotify) that executes the idea perfectly IMO, it's called Anghami, and here are pictures for examples.

The UI from the PC
![Anghami](https://user-images.githubusercontent.com/83197825/178496563-6ac1b90e-fb2f-4a9b-9c9c-b3fc2865691d.png)

The UI From the Phone
![image](https://user-images.githubusercontent.com/83197825/178496688-1ffc1657-cb97-429b-b6de-09620c20f7b0.png)

You should be able to control your playlist queue from your phone and vice versa like a remote.
as I see it very lacking from youtube themselves and other repos, again Idk about the possibility of implementing it, but if it were possible I think it'd be a great selling point.

Keep up the good work!

**Comment by Araxeus** [comment]
I haven't personally tested it nor do I endorse it, but a user claims to have created a fork that implements this feature

you can see his comment in this issue; https://github.com/th-ch/youtube-music/issues/771

implementing it in this app could be a good idea, but demands quite a lot of work which someone has to do, and I don't have much time nowadays 😅
